inventory.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Mender Software AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package main
+
+type InventoryAttribute struct {
+	Name  string      `json:"name"`
+	Value interface{} `json:"value"`
+}
+
+type InventoryData []InventoryAttribute
```

Comment:
Took another look at your proposal, I'm afraid it does not relate to what we want to do.
1. we're not reading files, but running external tools and fetching their output
2. we'd have to fork multiple `mender-inventory-*` tool instances and collect output from pipe of each - this complicates things via somewhat flaky pipe handling
3. InventoryData is passed directly to JSON encoder (in your case, we'd still need an intermediate type to conform to backend API, basically implement the whole 'conversion' just like it's done for `tempInventoryData`
4. it's not clear what you want to do with same key but produced by different tools

Thanks for reminding me about `strings.SplitN()` though. IMO, we should use it in identity data handling too.